src/network/networkprotocol.h

```diff
@@ -209,9 +209,11 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 		TOCLIENT_MEDIA_PUSH changed, TOSERVER_HAVE_MEDIA added
 		Added new particlespawner parameters
 		[scheduled bump for 5.6.0]
+	PROTOCOL VERSION 41:
```

Comment:
Apparently someone forgot to add documentation for version 41. This line should mention `PROTOCOL VERSION 42: `.

Reply:
[My fault](https://github.com/minetest/minetest/commit/c4277877b6a18939904900440b5e4accd69b11a8), should look like this I guess:
<pre>
	PROTOCOL VERSION 40:
		TOCLIENT_MEDIA_PUSH changed, TOSERVER_HAVE_MEDIA added
		Added new particlespawner parameters
	PROTOCOL VERSION 41:
		[scheduled bump for 5.6.0]
	PROTOCOL VERSION 42:
		Added support raillike tiles for 0 and 1 connections
</pre>